### DISCUSSION TITLE
Custom loader component for mux-video-react

### DISCUSSION THREAD

**Discussion by GuruPrakashDavala** [muxinc/elements#683] [Q&A]
Re: MuxVideo from mux-video-react package.

I would like to know if there is a prop to show a custom loader or spinner while the video is in the initial isLoading/isFetching state. 

Or is there a way to show only the loader control via CSS styles?

Thanks.

**Comment by gkatsev** [comment]
mux-video-react doesn't include any UI elements. All the potential UI for it comes from the video element. If you'd like more customized controls, you'd want to consider using mux-player-react instead.

**Reply by GuruPrakashDavala** [reply]
Okay. Thank you. Is there a way to programmatically know if the player is in a playing or loading (buffer) state?

For more information: I am building a swipeable reels video player where I disabled the mux video controls but I would like to show a loading spinner when the video is in a loading state.
